docs/integrations/saas-cloud/armis.md

```diff
@@ -193,16 +193,11 @@ This section explains how to collect logs from Armis API and ingest them into Su
 
 ## Installing the Armis App
 
-{@import ../../reuse/apps/app-install.md}
+{@import ../../reuse/apps/app-install-v2.md}
 
 ## Viewing the Armis Dashboards​
 
-* All dashboards have a set of filters that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.
-
- You can use filters to drill down and examine the data on a granular level. Filters include client country, client device type, client IP, client request host, client request URI, client request user agent, edge response status, origin IP, and origin response status.
-
-* Each panel has a set of filters that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.
-
+{@import ../../reuse/apps/view-dashboards.md}
 
 ### Alerts Overview Dashboard
```

Comment:
```suggestion
### Overview
```

Reply:
Please maintain the same for the other dashboard titles as well.

Reply:
Here, we need to keep it as **Alerts Overview Dashboard**, as this is the name for the dashboard. 
This app has two dashboards containing an overview in its title(Alerts Overview, Device Overview).